### PR TITLE
added kodadot to up-for-grabs

### PR DESCRIPTION
**Pull request by yangwao**
- We generally have issues available under the label "good first issue"
  - https://github.com/kodadot/nft-gallery/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22

- We are OSS with over **3000 raised pull requests and 144 contributors to this day.**
- [Check more at our activity page](https://github.com/kodadot/nft-gallery/blob/main/ACTIVITY.md)
- Contributors getting reward bounties to helping to solve them as it's a lot of times out of our current capacities.
- On top, long term contributors will be eligible for hire part-time/full-time based on the funding of the project.

### CODE REVIEW

File: _data/projects/kodadot.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+name: KodaDot - Polkadot NFT marketplace
+desc: open source implementation of nine NFT standards and growing to become multi-chain available nft marketplace
+site: http://github.com/kodadot/nft-gallery
+tags:
+- polkadot
+- vuejs
+- vue3
+upforgrabs:
+  name: good first issue
+  link: https://github.com/kodadot/nft-gallery/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22
```
Comment (shiftkey) [inline _data/projects/kodadot.yml]: :art: a simpler URL that will help with generating stats on our end

```suggestion
  link: https://github.com/ForNeVeR/Cesium/labels/good-first-issue
```

File: _data/projects/kodadot.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+name: KodaDot - Polkadot NFT marketplace
+desc: open source implementation of nine NFT standards and growing to become multi-chain available nft marketplace
+site: http://github.com/kodadot/nft-gallery
+tags:
+- polkadot
+- vuejs
+- vue3
+upforgrabs:
+  name: good first issue
+  link: https://github.com/ForNeVeR/Cesium/labels/good-first-issue
```
Comment (shiftkey) [inline _data/projects/kodadot.yml]:
```suggestion
  link: https://github.com/ForNeVeR/Cesium/labels/good%20first%20issue
```

File: _data/projects/kodadot.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+name: KodaDot - Polkadot NFT marketplace
+desc: open source implementation of nine NFT standards and growing to become multi-chain available nft marketplace
+site: http://github.com/kodadot/nft-gallery
+tags:
+- polkadot
+- vuejs
+- vue3
+upforgrabs:
+  name: good first issue
+  link: https://github.com/ForNeVeR/Cesium/labels/good%20first%20issue
```
Comment (shiftkey) [inline _data/projects/kodadot.yml]: 🤔 this should be a label in your repository, not the Cesium project

Comment (yangwao) [reply _data/projects/kodadot.yml]: Hey, you did this commit apparently by accident? https://github.com/up-for-grabs/up-for-grabs.net/pull/4108/commits/520cf215370eb9fbbbd1ee42b972017184a41cd4

**Comment by yangwao** [comment]
~~@shiftkey can you revert it?~~
never mind, changed it, lmk if it's ok!

😄
https://github.com/up-for-grabs/up-for-grabs.net/pull/4108/commits/520cf215370eb9fbbbd1ee42b972017184a41cd4

image

File: _data/projects/kodadot.yml (hunk 3 above)
Comment (yangwao) [inline _data/projects/kodadot.yml]:
```suggestion
  link: https://github.com/kodadot/nft-gallery/labels/good%20first%20issue
```

Comment (shiftkey) [reply _data/projects/kodadot.yml]: Oops, serves me right for trying to review multiple PRs at one. Apologies, thanks for repairing it!

**Review by shiftkey — APPROVED** [review]

**Pull request merged**